services/web/server/src/simcore_service_webserver/socketio/module_setup.py

```diff
@@ -0,0 +1,29 @@
+""" socketio subsystem based on socket-io
+    and https://github.com/miguelgrinberg/python-socketio
+
+"""
+import logging
+
+from aiohttp import web
+from socketio import AsyncServer
+
+from servicelib.application_setup import ModuleCategory, app_module_setup
+
+from . import handlers, handlers_utils
+from .config import APP_CLIENT_SOCKET_SERVER_KEY, assert_valid_config
+
+log = logging.getLogger(__name__)
+
+
+@app_module_setup("simcore_service_webserver.socketio", ModuleCategory.SYSTEM, logger=log)
+def setup_sockets(app: web.Application):
```

Comment:
would calling this `setup_socketio` be agreeable?

Reply:
yes